app/ide-desktop/lib/dashboard/globals.ts

```diff
@@ -0,0 +1,34 @@
+/** @file An object containing globals to inject. */
```

Comment:
I wouldn't know from docs here that this is about injecting _app configuration_.